suave/backends/transactional_store_test.go

```diff
@@ -0,0 +1,82 @@
+package backends
```

Comment:
Is this file here because of cycle dependency?

Reply:
Kind of, it relies on the local store